portal/README.md

```diff
@@ -1,32 +1,108 @@
 # Portal
 
+# Prerequisite
+
+1. Need to start `db` and `redis` container
+2. Need to start authgear servers
+
 # Setup locally
 
-We need to setup the environment variable `AUTHGEAR_CLIENT_ID=portal` and `AUTHGEAR_ENDPOINT=http://localhost:3000`.
+## Setup proxy
+
+We need a proxy server for us to call different server with the same domain / port. In `/portal` directory, there is a nginx container for proxy.
+
+To start proxy container
+
+```
+# in /portal
+docker-compose up -d
+```
+
+This container listen to port 8000. For redirection configuration, please refer to nginx config `/portal/nginx.conf`
+
+## Setup environment variable
+
+We need to setup the environment variable `AUTHGEAR_CLIENT_ID=portal` and `AUTHGEAR_ENDPOINT=http://localhost:3000`, and start graphQL server.
```

Comment:
I suppose we can also add these variables to .env.example for convenience  of setup.